### DISCUSSION TITLE
Welcome to museadon Discussions!

### DISCUSSION THREAD

**Discussion by emchateau** [ouvroir/museadon#68] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by emchateau** [comment]
Bonjour à tous, 
J’espère que vous allez bien et que vous passez une bonne rentrée. Ainsi que certains d’entre-vous s’en sont peut-être déjà aperçu, notre proposition de communication au Congrès de la SMQ a été acceptée. Je suis très heureux de l’intérêt qu’a suscité votre travail. 
J’ai demandé à la SMQ de pouvoir indiquer le nom des étudiants avec pour la communication mais c’est un peu compliqué car le congrès de la SMQ ne fonctionne pas comme un congrès scientifique habituel avec la publication du résumé accompagné de la liste des auteurs. J’ai eu plusieurs échanges avec les organisateurs et par ailleurs les modalités du panel ont changé. 
Initialement il s’agissait d’une table-ronde, finalement il s’agira seulement d’une présentation de 10min dans un panel dédié aux outils suivi d’une salle de réseautage en mode hybride. 
Je ne tiens pas particulièrement à faire la présentation et je trouverais préférable que deux d’entre vous qui seraient volontaires fassent cette présentation. Malheureusement, en 10min il me semble difficile de prévoir plus de personnes. Bien sûr je travaillerai étroitement avec vous pour la préparation du texte et la répétition. Il y a-t-il parmi vous des volontaires pour la présentation ? Je dois urgemment donner des noms aux organisateurs. La présentation aurait lieu le 14 octobre à 14h30 à Laval. Je vais également essayer de négocier un tarif pour l’entrée ou trouver une solution pour le prendre à ma charge. Merci beaucoup de vos réponses @ouvroir/museadon

**Reply by MaeliLC** [reply]
Bonjour à toutes et à tous ! C'est une très bonne nouvelle :) Je vais passer mon tour pour cette fois, j'ai confiance que n'importe qui saura bien nous représenter !

**Comment by franckcalard** [comment]
Je suis disponible à cette date, si vous êtes toujours à la recherche d'un volontaire je suis là ! @emchateau

**Reply by JuliettePassilly36** [reply]
Bonjour ! C'est une excellente nouvelle ! Je peux être disponible si besoin, même si je ne suis pas sure d'être la plus qualifiée pour la mission .. toutefois, si besoin, je suis là :)

**Comment by Alissia-CptBath** [comment]
Bonjour, Je suis disponible également!  Allons nous faire la présentation sur place ?  Par contre pour le déplacement à Laval, je n'ai pas de permis alors je ferais bien la route avec qqun ;) @ouvroir/museadon

**Reply by JuliettePassilly36** [reply]
Idem pour le déplacement :)

**Reply by emchateau** [reply]
Merci pour vos premières réponses. La présentation est en présence à Laval. 
Je n’ai pas non plus de permis mais on ne devrait pas avoir de mal à nous rendre à Laval. Il y a des transports en commun et au pire nous pourrions prendre un taxi.
Il faut que je propose deux noms. C’est encore ouvert si vous avez envie. On essaye de décider ensemble demain à la mi-journée ?

**Reply by franckcalard** [reply]
C'est parfait, est-ce que nous faisons une réunion zoom, sur un autre support ou via nos discussions sur Github ? Belle soirée à vous tous.

**Reply by emchateau** [reply]
Salut, je fais une présentation demain matin dans un séminaire à partir de 10h30 à l’UQAM. Il faut après que je participe à une réunion à 13h. Je peux être libre à partir de 14h si cela convient à certains d’entre vous. Sinon essayons de décider ici. Mais ce n’est pas très facile de se mettre d’accord avec un chat.

**Reply by franckcalard** [reply]
Je suis disponible demain pour 14h, attendons de voir la réponse des autres participants ? Il est vrai que le chat et les textes par réponses interposées ne facilitent pas les choses !

**Reply by Alissia-CptBath** [reply]
Il est vrai qu'il serait bien d'en discuter sur zoom. Cependant, j'ai un séminaire demain après midi entre 13h et 16h à l'UdeM, est il éventuellement possible de s'en parler plutôt en fin d'après midi (vers 17h? ) @ouvroir/museadon

**Reply by emchateau** [reply]
Pour moi c’est possible aussi à 17h.

**Reply by franckcalard** [reply]
Même chose de mon côté, on dit donc 17h ?

**Reply by Alissia-CptBath** [reply]
Parfait pour moi @ouvroir/museadon

**Reply by JuliettePassilly36** [reply]
Je ne serais disponible que très rapidement entre 16h30 et 16h45, je ne sais pas si ça peut être jouable pour vous ? Le Cas contraire, est ce que ce serait possible, que vous me fassiez un petit retour ?

**Reply by Alissia-CptBath** [reply]
Ou un autre moment pour que tout le monde soit présent, ou alors commencez avant et je vous rejoins @JuliettePassilly36 @ouvroir/museadon

**Reply by asibella** [reply]
Bonjour, 
Je serais très intéressée aussi. Serait-il envisage de pouvoir y assister en visio ? Je vais essayer de me rendre disponible pour ce soir (17h?) @ouvroir/museadon

**Reply by emchateau** [reply]
Voici un lien pour notre rencontre de 17h :
https://umontreal.zoom.us/j/99237515427?pwd=Uk1Qd3NoK2ljQkJ6QXkrTldhU05jdz09
Au plaisir de vous retrouver toutes et tous.
Emmanuel

**Reply by emchateau** [reply]
Au cas où vous n’auriez pas reçu l’invitation Voici un lien pour notre rencontre de 17h :
https://umontreal.zoom.us/j/99237515427?pwd=Uk1Qd3NoK2ljQkJ6QXkrTldhU05jdz09
Au plaisir de vous retrouver toutes et tous si vous en avez le temps @ouvroir/museadon 
Emmanuel
